README.md

```diff
@@ -56,33 +56,6 @@ that aren't) are directly accessible for use writing your own custom
 research tools. The interface should be nearly the same when dealing with
 a real process (via vdb/vtrace) and dealing with an emulator / viv workspace.
 
-## Installing
```

Comment:
I'm trying to keep the readme fairly light and move most things into the docs. To that, I need to go spruce up the readme again since I added the quickstart portions.